README.md

```diff
@@ -94,6 +94,7 @@ The Dockstore class in [src/app/shared/dockstore.model.ts](src/app/shared/dockst
 In `dockstore-webservice`, the `dockstore.yml` being served <b>must be edited to include the client IDs</b>.
 
 ## Pre-build/serve
+Run `export WEBSERVICE_VERSION=<dockstore release version>`, for example `export WEBSERVICE_VERSION=1.5.3`
```

Comment:
Maybe just link the version from https://github.com/dockstore/dockstore-ui2/blob/develop/.travis.yml#L12 so that the version stays current. Alternatively, there may be a simple Linux command to cut-out and get the version.
Something like `export WEBSERVICE_VERSION=`grep -oP 'WEBSERVICE_VERSION="\K[0-9].[0-9].[0-9]' .travis.yml`` but better in-case our version changes to include alpha/beta/rc (maybe look between the quotes)